Code/Framework/AzCore/Tests/Math/Vector2Tests.cpp

```diff
@@ -437,84 +437,84 @@ namespace UnitTest
         const float infinity = std::numeric_limits<float>::infinity();
         EXPECT_FALSE(AZ::Vector2(infinity, infinity).IsFinite());
     }
-    struct AngleTestArgs
+    struct Vector2AngleTestArgs
```

Comment:
there is a redefinition error but I didn't see it because these were in isolation ...